src/reader.c

```diff
@@ -127,10 +127,17 @@ static void *createStringObject(const redisReadTask *task, char *str, size_t len
 
     if (task->type == REDIS_REPLY_ERROR) {
         obj = createError(self->replyErrorClass, str, len);
+        if (obj == NULL) {
+            if (self->error.ptype == NULL)
+                PyErr_Fetch(&(self->error.ptype), &(self->error.pvalue),
+                        &(self->error.ptraceback));
+            obj = Py_None;
+            Py_INCREF(obj);
+        }
     } else {
         obj = createDecodedString(self, str, len);
     }
-
+    assert(obj != NULL);
```

Comment:
Why assert here? Take a look at line 134 we can know that obj could be NULL.

Reply:
True. Removed.